quipucords/scanner/network/processing/brms.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+
+"""Initial processing of raw shell output from Ansible commands."""
```

Comment:
Maybe add a more specific module comment if this is a copy & paste from eap.py.

Reply:
👍 I've got a change coming that is consistent with the string in the karaf file.